packages/components/space/style/index.less

```
@@ -0,0 +1,20 @@
+@import '../../style/default.less';
+@import "./mixins.less";
+
+@space-prefix: ~'@{idux-prefix}-space';
+@space-item-prefix: ~'@{space-prefix}-item';
+
+.@{space-prefix} {
+  display: inline-flex;
+
+  &-wrap {
+    flex-wrap: wrap;
+
+    .@{space-item-prefix} {
+      margin-bottom: @margin-md;
```

参考 button 组件, 需要在 default.less 中定义 `@space-item-margin-bottom: @margin-md;`

```suggestion
      margin-bottom: @space-item-margin-bottom;
```